components/feature/addons/src/test/java/mozilla/components/feature/addons/ui/AddonsManagerAdapterTest.kt

```diff
@@ -105,4 +111,92 @@ class AddonsManagerAdapterTest {
             }
         }
     }
+
+    @Test
+    fun bindAddon() {
+        Locale.setDefault(Locale.ENGLISH)
+        val titleView: TextView = mock()
+        val summaryView: TextView = mock()
+        val ratingAccessibleView: TextView = mock()
+        val userCountView: TextView = mock()
+        val addButton = ImageView(testContext)
+        val view = View(testContext)
+        val addonsManagerAdapterDelegate: AddonsManagerAdapterDelegate = mock()
+        val addonViewHolder = CustomViewHolder.AddonViewHolder(
+            view = view,
+            iconView = mock(),
+            titleView = titleView,
+            summaryView = summaryView,
+            ratingView = mock(),
+            ratingAccessibleView = ratingAccessibleView,
+            userCountView = userCountView,
+            addButton = addButton
+        )
+        val addon = Addon(
+            id = "id",
+            authors = emptyList(),
+            categories = emptyList(),
+            downloadUrl = "downloadUrl",
+            version = "version",
+            permissions = emptyList(),
+            rating = Addon.Rating(4.5f, 1000),
+            createdAt = "",
+            updatedAt = "",
+            translatableName = mapOf("en-US" to "name", "de" to "Name", "es" to "nombre"),
```

Comment:
nit: Should we also test when we don't get the translatable fields like `translatableName` and `translatableSummary` as they have a different paths.